drivers/gpio/gpio_stellaris.c

```diff
@@ -170,6 +175,93 @@ static int gpio_stellaris_read(struct device *dev,
 	return 0;
 }
 
+static int gpio_stellaris_port_get_raw(struct device *dev, u32_t *value)
+{
+	const struct gpio_stellaris_config *cfg = DEV_CFG(dev);
+	u32_t base = cfg->base;
+
+	*value = sys_read32(GPIO_RW_MASK_ADDR(base, GPIO_DATA_OFFSET, 0xff));
+
+	return 0;
+}
+
+static int gpio_stellaris_port_set_masked_raw(struct device *dev, u32_t mask,
+					 u32_t value)
+{
+	const struct gpio_stellaris_config *cfg = DEV_CFG(dev);
+	u32_t base = cfg->base;
+
+	sys_write32(GPIO_RW_MASK_ADDR(base, GPIO_DATA_OFFSET, mask), value);
```

Comment:
The parameters seem to be swapped, `value` should be first.

Reply:
Done.